src/components/trace/hat-graph-node.ts

```diff
@@ -1,6 +1,13 @@
-import { css, LitElement, svg } from "lit";
+import {
+  css,
+  LitElement,
+  PropertyValues,
+  html,
+  TemplateResult,
+  svg,
+} from "lit";
 import { customElement, property } from "lit/decorators";
-import { NODE_SIZE, SPACING } from "./hat-graph";
+import { NODE_SIZE, SPACING } from "./hat-script-graph";
```

Comment:
This feels like it's going the wrong way. hat-script-graph is using hat-graph-node and hat-graph-branch. They should not depend on it.
Especially if the plan is to use the graph otherwhere too later.

Reply:
Better to make them properties, maybe?